lib/Helper/Helper.cpp

```diff
@@ -12,7 +12,7 @@
 
 char* fgets_stdio_blocking(char* str, int n){
 
-    char c;
+    char c = '.';
```

Comment:
Why are you assigning a `.`? `c` is getting assigned/initialized in any case in the do body.

Reply:
You are right, removed the unnecessary initialization.